openedx/core/djangoapps/user_api/accounts/serializers.py

```diff
@@ -103,6 +104,11 @@ def to_representation(self, user):
             user_profile = None
             LOGGER.warning("user profile for the user [%s] does not exist", user.username)
 
+        try:
+            additional_fields = AdditionalRegistrationFields.objects.filter(user=user).first()
```

Comment:
The `exception` will not work here as well. kindly check the `filter().first()` method calls in Django ORM.